src/search3_rpc.erl

```diff
@@ -306,3 +306,53 @@ construct_bookmark_msg(_) ->
 
 get_channel() ->
     #{channel => ?SEARCH_CHANNEL}.
+
+set_update_sequence(Msg) ->
+    post("SetUpdateSequence", Msg, set_update_seq_request).
+
+delete_document(Msg) ->
+    post("DeleteDocument", Msg, document_delete_request).
+
+update_document(Msg) ->
+    post("UpdateDocument", Msg, document_update_request).
+
+search(Msg) ->
+    post("Search", Msg, search_request).
+
+group_search(Msg) ->
+    post("GroupSearch", Msg, group_search_request).
+
+info(Msg) ->
+    post("Info", Msg, index).
+
+delete_index(Msg) ->
+    post("Delete", Msg, index).
+
+analyze(Msg) ->
+    post("Analyze", Msg, analyze_request).
+
+post(Action, Request, RequestType) ->
+    EncodedRequest = search3_pb:encode_msg(Request, RequestType),
+    Endpoint = case application:get_env(search3, service_endpoint) of
+        {ok, E}
+            -> E;
```

Comment:
`E` here is a list of endpoints so this works by accident. You likely want to do a `get random item from E` here.

Reply:
wups, ya I did that.

Reply:
I messed this up, see @davisp's  comment below